.editorconfig

```diff
@@ -0,0 +1,6 @@
+root = true
```

Comment:
does this file want to be in here?

Reply:
Yeah it's some kind of standardized way of telling editors what settings to use for common things. Mine was indenting erb with 4 spaces, and figured this might be a good way of both fixing it and documenting it, even if people don't run the plugin. http://editorconfig.org/

Reply:
sneaky